src/SystemConfigGlobal.sol

```diff
@@ -4,8 +4,12 @@ pragma solidity ^0.8.0;
 
 import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import { ISemver } from "@eth-optimism-bedrock/src/universal/interfaces/ISemver.sol";
+import "@nitro-validator/src/INitroValidator.sol";
 
 contract SystemConfigGlobal is OwnableUpgradeable, ISemver {
+    /// @notice The AWS Nitro validator. 
+    INitroValidator public nitroValidator;
```

Comment:
this should be immutable (so that it's embedded in the bytecode and not in storage), as this contract is proxied